### PR TITLE
add copy kubectl and kubeconfig BAREMETAL

### PR DESCRIPTION
**Pull request by fanux**

### CODE REVIEW

File: runtime/utils.go [hunk 1]
```diff
@@ -76,3 +79,25 @@ func PreInitMaster0(sshClient ssh.Interface, remoteHostIP string) error {
...
 	return nil
 }
+
+func GetKubectlAndKubeconfig(ssh ssh.Interface, host string) error {
+	// fetch the cluster kubeconfig, and add /etc/hosts "EIP apiserver.cluster.local" so we can get the current cluster status later
+	err := ssh.Fetch(host, path.Join(common.DefaultKubeConfigDir(), "config"), common.KubeAdminConf)
+	if err != nil {
+		return errors.Wrap(err, "failed to copy kubeconfig")
+	}
+	err = utils.AppendFile(common.EtcHosts, fmt.Sprintf("%s %s", host, common.APIServerDomain))
```
Comment (cuisongliu) [inline runtime/utils.go:89]: need check etc host `common.APIServerDomain` is exist,some node this host config host is  localhost or vip

File: runtime/init.go [hunk 2]
```diff
@@ -63,9 +63,20 @@ func (d *Default) init(cluster *v1.Cluster) error {
 		return err
 	}
 
+	if err := d.GetKubectlAndKubeconfig(); err != nil {
```
Comment (cuisongliu) [inline runtime/init.go:66]: if master is changed. only at init step is not complete. I think the copy config need at all node change step.

File: runtime/utils.go (hunk 1 above)
Comment (fanux) [reply runtime/utils.go:89]: AppendFile already check the content is already exist, but not check the APIServerDomain, I will fix it in other PR

File: runtime/init.go (hunk 2 above)
Comment (fanux) [reply runtime/init.go:61]: If changed, User copy it by himself, If you copy it every time, it's inefficient.

**Review by cuisongliu — APPROVED** [review]

**Pull request merged**